File: app/src/main/java/com/cv4j/app/activity/ProjectHistActivity.java
package com.cv4j.app.activity;

import android.os.Bundle;
import android.support.design.widget.TabLayout;
import android.support.v4.app.Fragment;
import android.support.v4.view.ViewPager;
import android.support.v7.widget.Toolbar;

import com.cv4j.app.R;
import com.cv4j.app.adapter.ProjectHistAdapter;
import com.safframework.injectview.annotations.InjectExtra;
import com.safframework.injectview.annotations.InjectView;
import com.safframework.injectview.annotations.OnClick;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Tony Shen on 2017/6/4.
 */

public class ProjectHistActivity extends BaseActivity {

    @InjectView(R.id.tablayout)
    TabLayout mTabLayout;

    @InjectView(R.id.viewpager)
    ViewPager mViewPager;

    @InjectView(R.id.toolbar)
    Toolbar toolbar;

    @InjectExtra(key = "Title")
    String title;

    private List<Fragment> mList = new ArrayList();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_project_hist);

        initData();
    }

    private void initData() {
        toolbar.setTitle("< "+title);

        mList.add(new BackProjectFragment());
        mList.add(new GaussianBackFragment());
        mViewPager.setAdapter(new ProjectHistAdapter(this.getSupportFragmentManager(),mList));
        mTabLayout.setupWithViewPager(mViewPager);
    }

    @OnClick(id= R.id.toolbar)
    void clickToolbar() {

        finish();
    }
}

File: app/src/main/java/com/cv4j/app/activity/ProjectHistActivity.kt
package com.cv4j.app.activity

import android.os.Bundle
import androidx.fragment.app.Fragment
import com.cv4j.app.R
import com.cv4j.app.adapter.ProjectHistAdapter
import com.cv4j.app.app.BaseActivity
import com.cv4j.app.fragment.BackProjectFragment
import com.cv4j.app.fragment.GaussianBackFragment
import kotlinx.android.synthetic.main.activity_flip.*
import kotlinx.android.synthetic.main.activity_flip.toolbar
import kotlinx.android.synthetic.main.activity_project_hist.*
import java.util.*

/**
 *
 * @FileName:
 *          com.cv4j.app.activity.ProjectHistActivity
 * @author: Tony Shen
 * @date: 2020-05-05 21:00
 * @version: V1.0 <描述当前版本功能>
 */

class ProjectHistActivity : BaseActivity() {

    var title: String? = null

    private val mList: MutableList<Fragment> = ArrayList<Fragment>()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_project_hist)

        intent.extras?.let {
            title = it.getString("Title","")
        }?:{
            title = ""
        }()

        toolbar.setOnClickListener {
            finish()
        }

        initData()
    }

    private fun initData() {
        toolbar.title = "< $title"
        mList.add(BackProjectFragment())
        mList.add(GaussianBackFragment())
        viewpager.setAdapter(ProjectHistAdapter(this.getSupportFragmentManager(), mList))
        tablayout.setupWithViewPager(viewpager)
    }
}